src/components/RootApp/RootApp.tsx

```diff
@@ -77,7 +81,7 @@ const RootApp = (props: RootAppProps) => {
   };
   return (
     <Router history={chromeHistory}>
-      <SegmentProvider activeModule={activeModule}>
+      <SegmentProvider activeModule={activeModule!}>
```

Comment:
Interesting usage of non-null operator. Do I interpret this correctly as throw error is `activeModule` is null at build time?

Reply:
The module is just always string at this point. But it can be undefined in the redux store.